Tuist/Interfaces/SwiftUI/Project/Podfile

```diff
@@ -0,0 +1,57 @@
+platform :ios, '14.0'
```

Comment:
Can we make this file reusable for both interfaces, we only need to update the iOS version, right? 🙏

Reply:
The content are different, please not that we don't include RxSwift and UIKit related for SwiftUI.